src/run/docker-compose.ts

```diff
@@ -199,21 +201,23 @@ export class DockerCompose {
      * @throws {Error} If the AWS CLI command fails or credentials cannot be retrieved.
     */
     private get getAwsCredentials (): Record<string, string> {
+        if (!AWS_PROFILE || AWS_PROFILE === "undefined") {
```

Comment:
Will `AWS_PROFILE === "undefined"` ever hold true? Now it could be undefined or set - is this really an error case now?

Reply:
it could hold true if a user had a profile called 'undefined'. Its more of an additional check against invalid profile name and not just when its not set.

Reply:
it could hold true if a user had a profile called 'undefined'. Its more of an additional check against invalid profile name and not just when its not set.